src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -302,6 +303,23 @@ export default {
 			}
 		},
 	},
+
+	// TODO: move the implementation to @nextcloud-vue/NcListItem
+	watch: {
+		'item.displayName': {
+			immediate: true,
+			handler(value) {
+				this.$nextTick().then(() => {
+					const titleSpan = this.$refs.listItem?.$el?.querySelector('.line-one__title')
+
+					if (titleSpan && titleSpan.offsetWidth < titleSpan.scrollWidth) {
+						titleSpan.setAttribute('title', value)
```

Comment:
All this just to set the title attribute? It would be a minor change in the nextcloud-vue lib to add this there without the need for `watch` and `querySelector`.

Reply:
As I know, it is not possible to detect if a text fit container or not without putting it in the container and looking at the result in DOM. watch + DOM access could be missed only if a title is always shown independently from title length.

But a change in nextcloud-vue may help to avoid manual mutating of Vue-rendered DOM.